index.html

```diff
@@ -28,7 +28,19 @@ <h1>Welcome to the student showcase!</h1>
         <div>EIGHT</div>
         <div>NINE</div>
         <div>TEN</div>
-        <div>ELEVEN</div>
+        <div> ELEVEN</div>
+        <div>
+            <h2>Britten Sindelar</h2>
+            <img src= "https://pbs.twimg.com/profile_images/958715004584448002/45ipGSje.jpg">
+            <h3>Have Fun and Be yourself</h3>
+            <ul>
+             <li>
+             <i class="fas fa-paper-plane"></i>
```

Comment:
Remember that this only works if the font awesome library has been added to the page! You're free to add font awesome, though :D